.github/workflows/unit-tests.yml

```diff
@@ -34,6 +34,7 @@ jobs:
           - php: '8.3'
           - php: '8.4'
           - php: '8.5'
+            extensions: apcu,brotli,igbinary,intl,mbstring,memcached,redis
```

Comment:
why is this *adding* brotli ?

Reply:
I missed to remove it when switching the target branch from 7.3 to 6.4